src/main/java/org/jenkinsci/plugins/pipeline/milestone/MilestoneStep.java

```diff
@@ -118,20 +146,20 @@ public String getDisplayName() {
         @Override
         public void load() {
             super.load();
```

Comment:
this would appear to break upgrades where data was persisted as `milestonesByOrdinalByJob` but will be missing on restart as the map will be null.